app/_layouts/extension.html

```diff
@@ -216,13 +304,47 @@
 
 {{ plugin_config_for_route_admin_api }}
 
+`<route>` is the `id` or `name` of the Route that this plugin configuration
+  will target.
+
 {% else %}
 {% navtabs %}
 {% navtab Admin API %}
 
 {{ plugin_config_for_route_admin_api }}
 
+`<route>` is the `id` or `name` of the Route that this plugin configuration
+  will target.
+
 {% endnavtab %}
+{% unless page.params.manager_examples == false %}
+{% navtab Kong Manager %}
+Configure this plugin on a Route:
+
+1. In Kong Manager, select the workspace.
```

Comment:
4 steps for nav seems a bit much. Either something is off with the nav design, or the structure of the content. Suggest combining steps.

Reply:
All steps are required to get to where the user needs to be and each step is on a different page in the UI. I have combined where the steps are on the same page, but I do not think it is a good idea to combine steps that occur on different pages.